po_srvrmanager.py

```diff
@@ -13,51 +13,37 @@
 async def handle_clear_attendees(message: discord.Message):
     guild: discord.Guild = message.guild
     sroles = await get_roles(guild)
-    if sroles["✨Organizer"] in message.author.roles:
+    if sroles[po_roles.ORGANIZER_ROLE_ID] in message.author.roles:
+        new_event_helper = discord.utils.get(guild.members, id=NEW_EVENT_HELPER_USER_ID)
+        roles_to_clear = new_event_helper.roles[1:]
```

Comment:
If there's a specific role you want to include, we can't guarantee it'll always be the first one ( [1:] skips the first role in the list )

I'd suggest doing list(set(new_event_helper.roles[1:]) - ROLE_TO_IGNORE) instead